tests/kernel/wave/attention/quantized_attention_test.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2025 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import pytest
+import torch
+from iree.turbine.kernel.lang.global_symbols import *
+from iree.turbine.kernel.wave.utils.general_utils import (
+    get_default_scheduling_params,
+)
+from iree.turbine.kernel.wave.utils.run_utils import (
+    set_default_run_config,
+)
+from iree.turbine.kernel.wave.utils.torch_utils import (
+    device_randn,
+    device_zeros,
+)
+from iree.turbine.kernel.wave.compile import WaveCompileOptions, wave_compile
+from iree.turbine.kernel.wave.constraints import MMAType
+import os
+from torch.testing import assert_close
+from ..common.utils import (
+    require_e2e,
+    require_cdna3,
+    enable_scheduling_barriers,
+    dump_generated_mlir,
+    param_bool,
+)
+from ..common.shapes import get_test_shapes
+from iree.turbine.kernel.wave.templates.quantized_attention import (
+    get_brevitas_pertensor_fp8_attention_kernel,
+)
+from iree.turbine.kernel.wave.templates.attention_common import AttentionShape
+from iree.turbine.kernel.wave.scheduling.schedule import SchedulingType
+from iree.turbine.kernel.wave.compile import wave_compile, WaveCompileOptions
+
+test_quant_shape = [(1, 4096, 64, 64, 4096)]
+
+
+@require_e2e
+@require_cdna3
+@pytest.mark.parametrize("input_shape", test_quant_shape)
+@pytest.mark.parametrize("enable_scheduling", [SchedulingType.NONE])
+@param_bool("dynamic_dims", "dyn", [False])
+@pytest.mark.parametrize(
+    "mfma_variant",
+    [
+        (MMAType.F32_32x32x16_F8, MMAType.F32_32x32x16_K4_F8),
+        (MMAType.F32_16x16x32_F8, MMAType.F32_16x16x32_K4_F8),
+    ],
+)
+def testAttentionPure(
+    input_shape: tuple[int],
+    enable_scheduling: SchedulingType,
+    dynamic_dims: bool,
+    mfma_variant: tuple[MMAType],
+    request,
+):
+    run_bench = request.config.getoption("--runperf")
+    dump_perf = request.config.getoption("--dump-perf-files-path")
+    shape = AttentionShape(
+        num_query_heads=input_shape[0],
+        num_kv_heads=input_shape[0],
+        query_seq_len=input_shape[1],
+        head_size_kv=input_shape[2],
+        head_size=input_shape[3],
+        kv_seq_len=input_shape[4],
+    )
+    q_scale = 0.02578124962747097
```

Comment:
where do these numbers come from? Could you add a comment?

Reply:
sg!

Reply:
Done!